docs/apis/calcite.md

```diff
@@ -0,0 +1,570 @@
+Apache Apex, unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
```

Comment:
Although the different types of data has different processing needs, the SQL still remains a popular and  a generic way in which data processing can be represented.